### PR TITLE
Update user stories to match features

### PR DESCRIPTION
**Pull request by Professional-Procrastinat0r**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -364,7 +364,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   - 5a1. HRConnect shows an error message.
 
-    HRConnect displays the list of projects if the project id is invalid and the list of employees if the employee id is invalid. Use case resumes at step 4.
+    HRConnect displays the list of projects if the project id is invalid and the list of employees if the employee id is invalid. Use case resumes at step 5.
```
Comment (LeeZeHao) [inline docs/DeveloperGuide.md:367]: it does not automatically display the list of projects / employees, i think lets just change to "throws an error"

**Review by LeeZeHao — COMMENTED** [review]
LGTM in general, just this part is inaccurate!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Professional-Procrastinat0r) [reply docs/DeveloperGuide.md]: Makes sense, will updated

**Review by LeeZeHao — APPROVED** [review]
LGTM!

**Pull request merged**